conf/groups.conf.d/slurm.conf.example

```diff
@@ -12,6 +12,16 @@ all: sinfo -h -o "%N"
 list: sinfo -h -o "%R"
 reverse: sinfo -h -N -o "%R" -n $NODE
 
+#
+# SLURM node-reservation bindings
+#
+[slurmresv,sr]
+map: scontrol -o show reservation $GROUP | grep -Po 'Nodes=\K[^ ]+' | xargs | tr ' ' ','
```

Comment:
why `xargs | tr` ? This should only return 1 nodeset already?

Reply:
That's an excellent question! :)
I think I was trying to aggregate the returned nodeset on a single line in case of multi-line output (scripting habit) but you're right that it's not necessary here, since that command should return a single nodeset already. And I don't think it's necessary in the `all:` binding either. 
Fix incoming, thanks for catching that!